src/tiled/propertybrowser.cpp

```diff
@@ -947,15 +947,19 @@ QUndoCommand *PropertyBrowser::applyMapObjectValueTo(PropertyId id, const QVaria
         const bool flippedVertically = flippingFlags & 2;
 
         // You can only change one checkbox at a time
+        Cell newCell = mapObject->cell();
         if (mapObject->cell().flippedHorizontally() != flippedHorizontally) {
-            command = new FlipMapObjects(mMapDocument,
-                                         QList<MapObject*>() << mapObject,
-                                         FlipHorizontally);
+            newCell.setFlippedHorizontally(!mapObject->cell().flippedHorizontally());
```

Comment:
You don't need to toggle the existing value, please just do:
```c++
newCell.setFlippedHorizontally(flippedHorizontally);
```
You can even remove the condition and so you can assign the flipped state directly with `flippingFlags & 1` and `flippingFlags & 2`, since there is no longer a need for the additional variables regarding readability.